docs/providers/aws/guide/functions.md

```diff
@@ -211,6 +211,7 @@ provider:
     images:
       baseimage:
         path: ./path/to/context
+        file: Dockerfile.dev
```

Comment:
Does it make sense to have both `path` and `file`?

Wouldn't `file` make `path` obsolete? 

I wonder weather we could not rely just on `path`, and allow it to either point a directory that hosts `Dockerfile` or point `Dockerfile` specifically (?)

Reply:
I believe it does, for Docker `path` is translated to context `PATH` that is passed to `docker build`, where `file` is translated to `-f` which points to Dockerfile that needs to be used. By default, the `Dockerfile` is resolved with `path/Dockerfile`, but often `Dockerfiles` might reside in a separate directory, e.g. `docker`, where codebase is stored in `src`. In such case, `Dockerfile` won't be available in the provided context. In tools like `docker-compose`, these two are also separate options (`dockerfile` and `context`): https://docs.docker.com/compose/compose-file/compose-file-v3/#build  While in most cases we should be fine with just `file`, there are cases that require these two to be defined separately and for that reason, I would keep two separate options.

Reply:
Thanks for clarifications indeed it makes sense. Now I think that we should have named `path` a `context` (or `contextPath`), at least it'll pair better with `file`, but let's leave it for now

Reply:
I agree, `contextPath` would be probably the best name for that property